BlazarUI/app/scripts/components/sidebar/SidebarRepoList.jsx

```diff
@@ -21,16 +21,29 @@ class SidebarRepoList extends Component {
       );
     }
 
-    const buildsList = sortBuildsByRepoAndBranch(filterInactiveBuilds(filteredBuilds)).map( (build) => {
-      const buildType = has(build, 'inProgressBuild') ? 'inProgressBuild' : has(build, 'lastBuild') ? 'lastBuild' : 'neverBuilt'
+
+
+    const buildsList = Object.keys(filteredBuilds).sort().map((repo) => {
```

Comment:
this is turning into a hefty render method, I would extract some of this logic out of it.